glommio/src/task/raw.rs

```diff
@@ -365,6 +353,7 @@ where
     /// task reference to its schedule function.
     unsafe fn schedule(ptr: *const ()) {
         let raw = Self::from_ptr(ptr);
+        Self::increment_references(&mut *(raw.header as *mut Header));
 
         let guard;
         // Calling of schedule functions itself does not increment references,
```

Comment:
Is this still valid?

Reply:
yes, the whole point of the PR is to move the increment inside schedule, and take out the ones that happen in the callers